### DISCUSSION TITLE
Environment

### DISCUSSION THREAD

**Discussion by Abizrh** [3rd/image.nvim#242] [Q&A] [answered]
Does plugin only works using kitty and tmux? otherwise this plugin won't work?

**Comment by 3rd** [comment, marked as answer]
This plugin adds image support to Neovim using [Kitty's Graphics Protocol](https://sw.kovidgoyal.net/kitty/graphics-protocol/) or [ueberzugpp](https://github.com/jstkdng/ueberzugpp).
It works great with Kitty and Tmux, and it handles all the rendering complexity for you.

Tmux is optional
